src/providers/analytics/provider.ts

```diff
@@ -12,20 +12,30 @@ interface IndexAnalytics {
   symbol: string
   decimals: number
   address: string
-  totalSupply: string
-  marketPrice: number
-  navPrice: number
-  marketCap: number
-  change24h: number
-  volume24h: number
+  totalSupply: string | null
+  marketPrice: number | null
+  navPrice: number | null
+  marketCap: number | null
+  change24h: number | null
+  volume24h: number | null
+}
+
+interface IndexAnalyticsOptions {
+  includeTotalSupply?: boolean
+  include24hrVolume?: boolean
```

Comment:
Would it make sense while we are at it to make change24h and marketCap optional too?

Reply:
@janndriessen good call, added